test/empirical.jl

```diff
@@ -10,3 +11,9 @@ fnecdf = ecdf(randn(10000000))
 
 fnecdf = ecdf([0.5])
 @test fnecdf([zeros(5000); ones(5000)]) == [zeros(5000); ones(5000)]
+
+fnepdf = epdf(X)
+@test isapprox(fnepdf.([-10.0,0.0]), [0.0,0.383]; atol = 2e-3)
+
+fnepdf = epdf(X; nbins = 100)
+@test isapprox(fnepdf.([-10.0,0.0]), [0.0,0.398]; atol = 2e-3)
```

Comment:
I just copied the ECDF tests, where they use `isapprox`. The values depend on the sample `X`, so this would require that `X` is the same on all platforms etc. Does Julia generate the same pseudo-random numbers across different architectures?